src/Console/InstallCommand.php

```diff
@@ -52,7 +54,6 @@ protected function registerHorizonServiceProvider()
         $namespace = Str::replaceLast('\\', '', $this->laravel->getNamespace());
 
         if (file_exists($this->laravel->bootstrapPath('providers.php'))) {
-            // @phpstan-ignore-next-line
```

Comment:
PHPStan in CI was complaining that it didn't need this.

https://github.com/laravel/horizon/actions/runs/8335290386/job/22810504345